specification/purviewdatagovernance/PurviewDataCatalog/tspconfig.yaml

```
@@ -0,0 +1,44 @@
+parameters:
+  "service-dir":
+    default: "sdk/purviewdatagovernance"
+  "dependencies":
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-dir: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/Microsoft.Purview.UnifiedCatalog/{version-status}/{version}/CatalogApiService.json"
+    omit-unreachable-types: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-purview-unifiedcatalog"
+    namespace: "azure.purview.unifiedcatalog"
+    generate-test: true
+    generate-sample: true
+    flavor: azure
+  "@azure-typespec/http-client-csharp":
+    namespace: azure.purview.unifiedcatalog
```

```suggestion
    namespace: Azure.Purview.Unifiedcatalog
```
We need to update the generator to ignore the casing, but please change to the above.